tools/eslint-rules/timer-arguments.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * @fileoverview Require at least two arguments when calling setTimeout() or
+ * setInterval().
+ * @author Rich Trott
+ */
+'use strict';
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+function isTimer(name) {
+  return ['setTimeout', 'setInterval'].includes(name);
+}
+
+module.exports = function(context) {
+  return {
+    'CallExpression': function(node) {
+      const name = node.callee.name;
+      if (isTimer(name) && node.arguments.length < 2) {
+        context.report(node, `${name} must have exactly 2 arguments`);
```

Comment:
Yes, you are right, I updated the rule to accommodate more arguments but did not update the message. Fixed.